test/controllers/application_helper_test.rb

```diff
@@ -21,4 +21,39 @@ def test_add_args_to_url
     assert_equal("/blah\x80",
                  add_args_to_url("/blah\x80", x: "foo\xA0"))
   end
+
+  def test_textile_markup_should_be_escaped
```

Comment:
Why this test?  I try to avoid tests of third party tools since we have no control of them.